### DISCUSSION TITLE
Adding support for Mastodon Thread as a comment system for HTML

### DISCUSSION THREAD

**Discussion by cderv** [quarto-dev/quarto-cli#9779] [Feature Requests]
### Description

Opening to see if this gets interest or not. 

I found out about https://github.com/dpecos/mastodon-comments and it seems there is a way to make a Mastodon thread to be associated with a webpage, allowing this to be used as a public comment system. 

This is available as an extension by https://github.com/AndreasThinks/quarto-mastodon-comments by @AndreasThinks today

Quarto has integrated with different tools: https://quarto.org/docs/output-formats/html-basics.html#commenting

**Comment by AndreasThinks** [comment]
I obviously think this is an excellent idea :P

One challenge with the current plugin implementation is you need to post a "toot" _before_ doing the post (so you can then reference the tootId on the post metadata to pickup comments). It's not the smoothest flow, and in ideal world we'd have some fancy functionality to authenticate with your account, post a toot and grab the id.
